terra-app-setup-chart/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v2
 name: terra-app-setup
 description: Chart for set up entities for deploying Terra applications
 type: application
-version: 0.0.1
+version: 0.0.2
```

Comment:
I feel like it would be good to have a `CHANGELOG.md` in this repo

Reply:
or maybe just use `release`? we can create a release by doing something like
```
gh release create 0.0.2 -n "Bump log4cats-slf4j to 1.2.0\nBump fs2-core to 2.5.3"
```

Reply:
That sounds good too